plugins/hls-overloaded-record-dot-plugin/test/Main.hs

```diff
@@ -28,13 +42,36 @@ test = testGroup "overloaded-record-dot"
     mkTest "MultilineCase" "MultilineCase" "name" 10 7 12 15,
     mkTest "Multiline" "Multiline" "name" 10 7 11 15,
     mkTest "MultilineExpanded" "MultilineExpanded" "owner" 28 8 28 19
-  ]
+  ],
+  testGroup
+  "with Resolve"
```

Comment:
I'd be tempted to reverse the nesting, i.e. define the test case as a group and then do a resolve/non-resolve test inside it. Should let you avoid the duplication of the test specs.